resources/js/views/settings/roomTypes/View.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+  <b-overlay :show="isBusy || !loaded">
+
+    <template #overlay>
+      <div class="text-center">
+        <b-spinner v-if="isBusy" ></b-spinner>
+        <b-button
+          ref="reloadRoomType"
+          v-else
+          @click="loadRoomType()"
+        >
+          <b-icon-arrow-clockwise></b-icon-arrow-clockwise> {{ $t('app.reload') }}
+        </b-button>
+      </div>
+    </template>
+
+    <h3>
+      {{ id === 'new' ? $t('settings.roomTypes.new') : (
+        viewOnly ? $t('settings.roomTypes.view', { name: model.description })
+          : $t('settings.roomTypes.edit', { name: model.description })
+      ) }}
+    </h3>
+    <hr>
+
+    <b-form @submit='saveRoomType'>
+      <b-container fluid>
+        <b-form-group
+          label-cols-sm='4'
+          :label="$t('settings.roomTypes.description')"
+          label-for='description'
+          :state='fieldState("description")'
+        >
+          <b-form-input id='description' type='text' v-model='model.description' :state='fieldState("description")' :disabled='isBusy || viewOnly'></b-form-input>
```

Comment:
The inputs should also be disabled if there was an loading error, since otherwise you could use tabs to edit the input.